tst/EnergyPlus/unit/ZoneEquipmentManager.unit.cc

```diff
@@ -3833,7 +3833,7 @@ TEST_F(EnergyPlusFixture, ZoneAirMassFlowBalance_wAdjustMixingThenReturn)
     GetProjectControlData(*state, ErrorsFound);
     EXPECT_FALSE(ErrorsFound);
     EXPECT_TRUE(DataHeatBalance::ZoneAirMassFlow.EnforceZoneMassBalance);
-    EXPECT_EQ(DataHeatBalance::ZoneAirMassFlow.ZoneFlowAdjustment, DataHeatBalance::AdjustMixingThenReturn);
+    EXPECT_EQ(DataHeatBalance::ZoneAirMassFlow.ZoneFlowAdjustment, DataHeatBalance::AdjustmentType::AdjustMixingThenReturn);
     EXPECT_EQ(DataHeatBalance::ZoneAirMassFlow.InfiltrationTreatment, DataHeatBalance::AdjustInfiltrationFlow);
     EXPECT_EQ(DataHeatBalance::ZoneAirMassFlow.InfiltrationZoneType, DataHeatBalance::AllZones);
     GetSimpleAirModelInputs(*state, ErrorsFound);
```

Comment:
Various update of unit test results that checks the zone air flow balancing methods. Now the code uses Enum class instead of "int" parameters.